keyboards/planck/ez/rules.mk

```diff
@@ -24,10 +24,8 @@ API_SYSEX_ENABLE = no
 SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
 #SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
 
-CUSTOM_MATRIX = no # Custom matrix file
 # SERIAL_LINK_ENABLE = yes
 ENCODER_ENABLE = yes
-RGB_MATRIX_ENABLE = IS31FL3737
```

Comment:
As above, but no idea if default works this deep
`DEFAULT_FOLDER = planck/ez/glow`

Reply:
It does, actually.  

But part of the reason for the glow folder is that ... if you don't have the hardware (the ISSI controller) and you have rgb matrix enabled, it makes the board nigh unusable.  It's an issue that has come up multiple times on the discord, so ... defaulting to it is less than ideal (eg, we shouldn't, outright).  Would rather default to non-glow model, again, as the ErgoDox EZ Configurator will include the rgb matrix setting.